.travis.yml

```diff
@@ -2,8 +2,8 @@ language: java
 
 env:
   global:
-    - MAVEN_OPTS="-Xmx256M"
-    - MAVEN_SKIP_CHECKS_AND_DOCS="-Dair.check.skip-all=true -Dmaven.javadoc.skip=true"
```

Comment:
We do not quote the matrix variables. So, just to keep the same here. I can remove this commit though.